errors/__tests__/apiErrors.ts

```diff
@@ -23,33 +24,32 @@ export const newError = (overrides = {}): BaseError => {
 };
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-export const newAxiosError = (overrides: any = {}): AxiosError => {
-  return new AxiosError(
-    '',
-    overrides.code || '',
-    { headers: new AxiosHeaders(), ...overrides.config },
-    {
-      ...overrides.request,
-    },
-    {
-      request: {
-        href: 'http://example.com/',
-        method: 'GET',
+export const newHubSpotHttpError = (overrides: any = {}): HubSpotHttpError => {
+  return new HubSpotHttpError('', {
+    cause: {
+      ...newError(),
+      isAxiosError: true,
+      name: 'HubSpotHttpError',
+      response: {
+        request: {
+          href: 'http://example.com/',
+          method: 'GET',
+        },
+        data: {},
+        headers: {},
+        status: 200,
```

Comment:
Nit: Do we want another 400/500 status code?

Reply:
Good eye, this was carryover, but I agree it feels weird for an error to be 200.  All of the tests appear to be overriding it to test the specific cases, so I removed the status from this object.